src/components/Navigation/index.tsx

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+import styles from  './Navigation.module.css';
+import { RootState } from "../../store";
+import { AppDispatch } from "../../store";
+import { fetchAllBlogs, nextPage, prevPage, showCurrentPage } from "../../slices/blogSlice";
+import { connect,ConnectedProps } from "react-redux";
+import { fetchBlogs } from "../../slices/blogSlice";
+
+const mapStateToProps = (state: RootState) => {
+  return {
+    fullArrayLength: state.blogs.fullArrayLength,
```

Comment:
Am I right, you fetch all 5000 items just to count them?

Reply:
API doesn't have `total` in response?

Reply:
note: some servers add `X-Total-Count` (or smth like that) headers with count of all items